swtbot_test/output/ConfigurationErrorCheckFile.txt

```diff
@@ -0,0 +1,12 @@
+Project iot_adu_GCC_ckitrx65n has configuration error on component: r_fwup(configuration error)
```

Comment:
Q: This log has many errors, is the test correct? or we ignore this log?

Reply:
We ignore this log